packages/components/src/RecurringSelect/DateCellCheckbox.css

```diff
@@ -0,0 +1,80 @@
+.container {
+  display: block;
+}
+
+.checkboxWrapper {
+  position: relative;
+}
+
+/* kept present but hidden for accessibility */
+.container input[type="checkbox"] {
```

Comment:
hmm. can we add a class name on that input instead of nested styling?

Reply:
possibly, yes

however if, like the rest of the style changes I've been making in here, it turns out to be a huge pain

I'm leaving to become a farmer or something

Reply:
🧑‍🌾